keyboards/handwired/aetreus/keymaps/janzon/keymap.c

```diff
@@ -0,0 +1,238 @@
+ *
+ * Enter becomes shift when held down
+ * Tab goes to SYM layer when held down
+ * Alt becomes Tab when tapped (only used for Cmd + Tab)
+ * , . / becomes < > ? when shift is pressed
+ * HJKL becomes arrows when right shift is pressed
+ *
+ */
+
+[_QWERTY] = LAYOUT(
+    KC_MYTAB,    KC_Q,    KC_W,     KC_E,    KC_R,      KC_T,        KC_Y,     KC_U,     KC_I,    KC_O,      KC_P, KC_ARING, \
+      KC_ESC,    KC_A,    KC_S,     KC_D,    KC_F,      KC_G,        KC_H,     KC_J,     KC_K,    KC_L,   KC_OUML,  KC_AUML, \
+     _______,    KC_Z,    KC_X,     KC_C,    KC_V,      KC_B,        KC_N,     KC_M,  ANSICOM, ANSIDOT,   ANSIKEY, _______, \
+                       KC_LSFT, KC_MYGUI, KC_MYSYM, KC_MYSFT,     KC_SPC, KC_MYCTL, KC_MYALT, _______                       \
+),
+
+/* SYM
+ * ,-----------------------------------------.   ,-----------------------------------------.
+ * |   `  |   !  |   @  |   {  |   }  |   &  |   |   *  |   7  |   8  |   9  |   +  |   =  |
+ * |------+------+------+------+------+-------   |------+------+------+------+------+------|
+ * |   ~  |   ^  |   $  |   (  |   )  |   '  |   |   ;  |   4  |   5  |   6  |   -  |   _  |
+ * |------+------+------+------+------+------|   |------+------+------+------+------+------|
+ * |   |  |   #  |   %  |   [  |   ]  |   "  |   |   :  |   1  |   2  |   3  |   \  |  =>  |
+ * |------+------+------+------+------+------+   |------+------+------+------+------+------|
+ * |      |      | RESET|      |      |      |   |      | C_ENT|   0  |      |      |      |
+ * `-----------------------------------------'   `-----------------------------------------'
+ */
+[_SYM] = LAYOUT( \
+    KC_SE_GRAV,    KC_EXLM,  KC_SE_AT, KC_SE_LCBR, KC_SE_RCBR, KC_SE_AMPR, KC_SE_ASTR,    KC_7,      KC_8,    KC_9, KC_SE_PLUS,  KC_SE_EQL, \
+    KC_SE_TILD, KC_SE_CIRC, KC_SE_DLR, KC_SE_LPRN, KC_SE_RPRN, KC_SE_QUOT, KC_SE_SCLN,    KC_4,      KC_5,    KC_6, KC_SE_MINS, KC_SE_UNDS, \
+    KC_SE_PIPE,    KC_HASH,   KC_PERC, KC_SE_LBRC, KC_SE_RBRC, KC_SE_DQUO, KC_SE_COLN,    KC_1,      KC_2,    KC_3, KC_SE_BSLS,    _______, \
+                                RESET,    _______,    _______,    _______,    _______, KC_CENT,      KC_0, _______                          \
+),
+
+/* Sys
+ */
+
+[_SYS] = LAYOUT(
+    KC_MYTAB, KC_VOLD, KC_MUTE, KC_VOLU,       KC_FFCT, _______,        _______, KC_SDTG, KC_SSP1, KC_SSP2, KC_SSP3, _______, \
+   _______, KC_MRWD, KC_MPLY, KC_MFFD, MY_SCREENSHOT, _______,        KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT, _______, _______, \
+   _______, _______, _______, _______,       _______, _______,        _______, KC_DTOG, KC_SPC1, KC_SPC2, KC_SPC3, _______, \
+                         RESET, _______,       _______, _______,        _______, _______, _______, _______                   \
+)
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer